lake/modules/sync_groups.py

```diff
@@ -233,4 +233,4 @@ def next_gate_mask(self):
 if __name__ == "__main__":
     db_dut = SyncGroups(fetch_width=64,
                         int_out_ports=2)
-    verilog(db_dut, filename="sync_groups.sv", optimize_if=False)
+    verilog(db_dut, filename="sync_groups.sv", optimize_if=False, check_flip_flop_always_ff=False)
```

Comment:
delete check_flip_flop_always_ff=False

Reply:
we will eventually - there could still be issues without the flags so for now I'd like to keep it